### PR TITLE
Change tab bar style

### PR DESCRIPTION
**Pull request by TheKnightGamer**

### CODE REVIEW

File: components/Layout.jsx [hunk 1]
```diff
@@ -18,6 +18,20 @@ const Layout = ({ title, children }) => {
 
   return (
     <>
+      <style type="text/css">
```
Comment (nathfreder) [inline components/Layout.jsx]: this should be in a scss file

File: components/Layout.jsx [hunk 2]
```diff
@@ -18,6 +18,20 @@ const Layout = ({ title, children }) => {
...
+        {`
+      .list-group-item {
+         position: relative;
+         display: block;
+         padding: .75rem 1.25rem;
+         border: 1px solid;
+         border-color: black;
```
Comment (nathfreder) [inline components/Layout.jsx]: these don't add anything... idk why they exist
i'm probably confused so if you can clarify

File: components/Layout.jsx [hunk 3]
```diff
@@ -18,6 +18,20 @@ const Layout = ({ title, children }) => {
...
+         border-radius: 20px;
```
Comment (nathfreder) [inline components/Layout.jsx]: i would suggest `50rem` but that makes everything look strange

File: components/Layout.jsx [hunk 4]
```diff
@@ -18,6 +18,20 @@ const Layout = ({ title, children }) => {
...
+         margin: 5px;
```
Comment (nathfreder) [inline components/Layout.jsx]: this shouldn't exist

File: components/Layout.jsx (hunk 2 above)
Comment (TheKnightGamer) [reply components/Layout.jsx]: well I just added all of it to be on the safe side didn't want any errors

File: components/Layout.jsx (hunk 4 above)
Comment (TheKnightGamer) [reply components/Layout.jsx]: yes that make the pill shaped tab

**Comment by nathfreder** [comment]
I think the rules are inherited. I’ll check when I get home.

**Pull request closed without merging**